### ISSUE TITLE
implementing onBackPressed for SlidingLayer

### ISSUE THREAD

**Issue by pishguy** [wunderlist/android-sliding-layer-lib#109]
i have a big problem with this library such as implementing onBackPressed for opened SlidingLayers on fragments or activities, how can i do that without implementing android listeners to detect that?

**Comment by Mr-Krabbe** [comment]
What I did to prevent back navigation when SlidingLayer is open is to simple override onBackPressed() and check if the layer is open. Was this your question?

```java
    @Override
    public void onBackPressed() {
        if (slidingLayer.isOpened()) {
            slidingLayer.closeLayer(true);
        } else {
            super.onBackPressed();
        }
    }
```

**Comment by JlUgia** [comment]
Right, that's left at your complete control and will. Where you able to find a similar solution to apply to your navigation patterns @MahdiPishguy ?
